src/internal/hooks/use-virtual/index.ts

```diff
@@ -16,11 +16,13 @@ interface UseVirtualProps<Item> {
   parentRef: React.RefObject<HTMLElement>;
   estimateSize: () => number;
   firstItemSticky?: boolean;
+  applyItemOffset?: boolean;
```

Comment:
Should we call this item overlap instead of item offset? We can also consider using `itemOverlap: number` so that it is better explicit and better flexible.